plugins/domains/scribd.com/scribd.com-error.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+
+    re: [
+        /^https?:\/\/(?:www|\w{2})\.scribd\.com\/(doc|document|embeds|presentation|fullscreen)\/(\d+)/i
+    ],
+
+    mixins: [
+        "oembed-title",
+        "oembed-thumbnail",
+        "oembed-site",
+        "oembed-author",
+        "og-image",
+        "domain-icon",
+        "og-description"
+    ],
+
+    getData: function(oembedError, options, cb) {
+        if (oembedError === 401){
+            cb({
+                responseStatusCode: 401,
```

Comment:
No need to 401 on the link entirely. It's a valid URL, just not embeddable. We can still create a summary card for it. 
A message is enough.